engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -576,6 +576,11 @@ public void startApplication() throws YarnException, IOException
         vargs.add("-Dlog4j.debug=true");
       }
 
+      String loggerAppender = dag.getValue(LogicalPlan.LOGGER_APPENDER);
```

Comment:
The property -Dapex.logger.appender is set for the client. And the property -Ddt.attr.LOGGER_APPENDER is set for the application. We cannot use the same property if we need to support the scenario: to define the appender for the client and not to define the appender for the application.